docs/feature_combo.md

```diff
@@ -101,3 +189,7 @@ In addition to the keycodes, there are a few functions that you can use to set t
 | `combo_disable()`    | Disables the combo feature, and clears the combo buffer |
 | `combo_toggle()`     | Toggles the state of the combo feature                  |
 | `is_combo_enabled()` | Returns the status of the combo feature state (true or false) |
+
+
+#### Dictionary Management
+Combos can be augmented by the [Combo Engine](http://combos.gboards.ca/) to allow for easier configuration and multifile dictionaries. When a large number of Combos are present, these decorators allow for simpler management and definition of Combos. With single line combo definitions and various helpers consider using this when over 20 combos are present on a keyboard. Under the hood, the Combo Feature is used and all the above configuration will work with it.
```

Comment:
« [....] these decorators allow for simpler management and definition of Combos. With single line combo definitions and various helpers consider using this when [...] » → « [...] these decorators allow for simpler management and definition of Combos with single line combo definitions and various helpers. Consider using this when [...] »